src/overlays/actors/ovl_En_Bom/z_en_bom.c

```diff
@@ -315,8 +315,11 @@ void EnBom_Update(Actor* thisx, GlobalContext* globalCtx2) {
             }
 
             Audio_PlayActorSound2(thisx, NA_SE_IT_BOMB_EXPLOSION);
-            globalCtx->envCtx.unk_8C[1][0] = globalCtx->envCtx.unk_8C[1][1] = globalCtx->envCtx.unk_8C[1][2] = 0xFA;
-            globalCtx->envCtx.unk_8C[0][0] = globalCtx->envCtx.unk_8C[0][1] = globalCtx->envCtx.unk_8C[0][2] = 0xFA;
+            if (globalCtx) {};
```

Comment:
Is this necessary now? If so, wouldn't it indicate that the 2d array is the right thing to have?

Reply:
its fine without. i believe this is old and was added back (had a git mishap at some point)